js/base/Exchange.js

```diff
@@ -1685,6 +1686,17 @@ module.exports = class Exchange {
          * @param {string|undefined} currencyCode unified currency code, but this argument is not required by default, unless there is an exchange (like huobi) that needs an override of the method to be able to pass currencyCode argument additionally
          * @returns {[string|undefined]} exchange-specific network id
          */
+        if (this.options['hasUniqueNetworkIds']) {
```

Comment:
so, this change only affects if exchange has `hasUniqueNetworkIds` to true in options. thus, this change doesn't affect any existing implementations (other than HUOBI & OKX)

Reply:
> @ttodua this option does not seem very "self-explanatory", what if instead of the option we checked the size of networkChainIdsByNames/networkNamesByChainIds ? If bigger than 0, means we have this logic, wdyt?

I'll reply.